eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -20,7 +20,39 @@ function Get-javascript-OnboardedDocsMsPackages($DocRepoLocation) {
   return $onboardedPackages
 }
 
-function Get-javascript-DocsMsTocData($packageMetadata, $docRepoLocation) {
+function Get-javascript-OnboardedDocsMsPackagesForMoniker($DocRepoLocation, $moniker) {
+  $packageOnboardingFile = ""
+  if ("latest" -eq $moniker) {
+    $packageOnboardingFile = "$DocRepoLocation/ci-configs/packages-latest.json"
+  }
+  if ("preview" -eq $moniker) {
+    $packageOnboardingFile = "$DocRepoLocation/ci-configs/packages-preview.json"
+  }
+
+  $onboardedPackages = @{}
+  $onboardingSpec = ConvertFrom-Json (Get-Content $packageOnboardingFile -Raw)
+  foreach ($spec in $onboardingSpec.npm_package_sources) {
+    $packageName = $spec.name
+
+    if ($packageName.LastIndexOf('@') -gt 0) {
```

Comment:
I think it's OK to not refactor this logic at this time because we might remove `Get-javascript-OnboardedDocsMsPackages` in favor of this per-moniker approach leaving only one such place for this code (that my search of the repository turned up)